ui/selectors/selectors.js

```diff
@@ -82,8 +82,9 @@ export function getCurrentKeyring(state) {
 }
 
 export function isEIP1559Account(state) {
-  // Neither hardware wallet supports 1559 at this time
-  return !isHardwareWallet(state);
+  // Only Ledger supports 1559 at this times
```

Comment:
Nit: I assume this meant "Among our supported hardware wallets, only Ledger supports 1559". This code is run for local accounts too after all. Maybe this wording would be clearer:

```suggestion
  // Trezor does not support 1559 at this time
```